src/components/MovieCard/MovieCard.jsx

```diff
@@ -1,5 +1,50 @@
 import './MovieCard.scss';
 
-export const MovieCard = () => (
-  <>Put the card here</>
+export const MovieCard = ({
+  movie:
+  { imgUrl,
+    title,
+    description,
+    imdbUrl },
+}) => (
+  <div className="card" data-cy="Movie">
+    <div className="card-image">
+      <figure className="image is-4by3">
+        <img
+          data-cy="MovieImage"
+          src={imgUrl}
+          alt={`Film logo of ${title}`}
+        />
+      </figure>
+    </div>
+
+    <div className="card-content">
+      <div className="media">
+        <div className="media-left">
+          <figure className="image is-48x48">
+            <img src="images/imdb-logo.jpeg" alt="imdb" />
+          </figure>
+        </div>
+
+        <div className="media-content">
+          <p className="title is-8" data-cy="MovieTitle">
+            {title}
+          </p>
+        </div>
+      </div>
+
+      <div className="content">
+        <p data-cy="MovieDescription">
+          {description}
+        </p>
+
+        <a
+          href={imdbUrl}
+          data-cy="MovieLink"
+        >
+          IMDB
+        </a>
```

Comment:
Consider adding target='_blank' and rel='noopener noreferrer' attributes to the anchor tag to open the link in a new tab and for security reasons.